ansible/roles/ocp4-workload-summit-api-cicd/tasks/remove_workload.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: virtualenv path to use for pip
+  set_fact:
+    pip_path: "{{ ansible_python_interpreter | dirname }}"
+
+- name: "Uninstall lxml module from python"
+  command: "{{ pip_path }}/pip uninstall lxml -y"
+  become: True
+
+- name: "Remove {{ API_MANAGER_NS }} Project"
+  shell: "oc delete project {{API_MANAGER_NS}}"
```

Comment:
Would be better to do a k8s loop:

```
k8s:
  state: absent
  api_version: project.openshift.io/v1
  kind: Project
  name: "{{ item }}"
loop:
- "{{ DEV_GW_PROJECT }}"
- ....
```

Reply:
Thanks @wkulhanek . I changed this to use k8s. Will send another pull request to address it.